src/connectivity.jl

```diff
@@ -518,19 +518,20 @@ function _neighborhood(g::AbstractGraph{T}, v::Integer, d::Real, distmx::Abstrac
         seen[src] && continue
         seen[src] = true
         currdist = dists[src]
+        push!(neighs, src)
         vertexneighbors = neighborfn(g, src)
         @inbounds for vertex in vertexneighbors
             if !seen[vertex] 
-                vdist = currdist + distmx[src, vertex]
-                if vdist <= d
+                vdist = currdist == typemax(U) ? typemax(U) : currdist + distmx[src, vertex]
```

Comment:
If currdist == typemax(U) then vdist cannot be < dists[vertex] , so shouldn't we just shift the check currdist == typemax(U) outside the loop and continue if it is true ? Or am I missing something here ?

Reply:
Nope, you're absolutely right. Fixing this now.